spaces/S000136/properties/P000088.md

```diff
@@ -0,0 +1,15 @@
+---
+uid: XXXX
```

Comment:
Not sure how to chose uid for a property file within a space.  What's the guideline here?